packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -425,6 +432,65 @@ export class StripeWebhookHandler extends StripeHandler {
     await this.updateCustomer(uid, email);
   }
 
+  /**
+   * Validate plan metadata on update
+   */
+  async handlePlanUpdatedEvent(request: AuthRequest, event: Stripe.Event) {
+    const plan = event.data.object as Stripe.Plan;
+    const product = await this.stripeHelper.fetchProductById(
+      plan.product as string
+    );
+    if (!product) {
+      this.log.error(
+        `handlePlanUpdatedEvent - Could not locate Product ${plan.product}`,
+        { plan }
+      );
```

Comment:
This error is unlikely but maybe also report to Sentry if it does happen?  Otherwise we won't know about it until someone reports a problem.

Reply:
updated to log to sentry and adjusted corresponding test